README.md

```diff
@@ -190,3 +190,33 @@ When given a command, Open Crawler will run until the process is finished.
+|--------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|-------------------------|
+| **Interface**                                                                                                                                          | CLI          | GUI (Kibana)            |
+| **Hosting**                                                                                                                                            | Self-hosted  | Elastic Cloud           |
+| **State management**                                                                                                                                   | Stateless    | Elasticsearch indices   |
+| **Compatible with [Elasticsearch Serverless](https://www.elastic.co/docs/current/serverless)**                                                         | Yes          | No                      |
+| **Unrestricted index naming**                                                                                                                          | Yes          | No                      |
+| **Indexing using `_bulk` API**                                                                                                                         | Yes          | No                      |
+| **[Ingest pipelines](./docs/features/INGEST_PIPELINES.md)**                                                                                            | Yes          | Yes                     |
+| **[Binary content extraction](./docs/features/BINARY_CONTENT_EXTRACTION.md)**                                                                          | Yes          | Yes                     |
+| **[Crawler directives](./docs/features/CRAWLER_DIRECTIVES.md)**                                                                                        | Yes *        | Yes                     |
+| **[Scheduling](#scheduling-recurring-crawl-jobs)**                                                                                                     | Yes          | Yes                     |
+| **Crawler results history and metadata**                                                                                                               | No **        | Yes                     |
+| **[Duplicate content handling](https://www.elastic.co/guide/en/enterprise-search/current/crawler-managing.html#crawler-managing-duplicate-documents)** | No           | Yes                     |
+| **Event Logging**                                                                                                                                      | No ***       | Yes                     |
+| **Monitoring**                                                                                                                                         | No           | Yes                     |
```

Comment:
This feels redundant to the `Event Logging` and `Crawl result history and metadata` rows